app/graphql/types/credit_notes/object.rb

```diff
@@ -37,13 +37,24 @@ class Object < Types::BaseObject
 
       field :file_url, String, null: true
 
+      field :customer, Types::Customers::Object, null: false
       field :invoice, Types::Invoices::Object
       field :items, [Types::CreditNoteItems::Object], null: false
 
+      field :is_credited, Boolean, null: false
+      field :is_refunded, Boolean, null: false
```

Comment:
Do you have a need for these fields?

The methods they rely on are defined like this:

```ruby
  def credited?
    credit_amount_cents.positive?
  end

  def refunded?
    refund_amount_cents.positive?
  end
```

and both `refund_amount_cents` and `credit_amount_cents` are already exposed

Reply:
I could remove them indeed ! Done